RIS.js

```diff
@@ -1266,6 +1268,22 @@ function processTag(item, tagValue, risEntry) {
 				zField = ['extra'];
 			}
 		break;
+		case "H1":
+			//H1, H2 can have multiple occurences which cannot be saved
+			if(item.libraryCatalog) {
+				item.multipleLibraries = value;
+				value = '';
+			}
+		break;
+		case "H2":
+			//H1, H2 can have multiple occurences which cannot be saved
+			if(item.callNumber) {
+				zField = ['unsupported', 'H1/H2'];
+				if(item.multipleLibraries) {
+					value = item.multipleLibraries+": "+value;
```

Comment:
idk how much you care, but this doesn't work very well if you have things like H1, H2, H2; H1, H1, H2; H1 only; H2 only. I don't particularly care if we lose or make a bit of a mess of some of these values (I'd prefer if we did things right of course). This would suggest a messed up RIS anyway.